app/views/transferSummary.scala.html

```diff
@@ -45,19 +45,19 @@ <h1 class="govuk-heading-xl">@Messages("transferSummary.title")</h1>
                 <fieldset class="govuk-fieldset" aria-describedby="transfer-summary">
                     <!-- Open records & transfer legal ownership -->
                     @inputSingleCheckbox(
-                    transferSummaryForm("openRecords"),
+                    transferConfirmationForm("openRecords"),
                     Symbol("_label") -> Messages("transferSummary.openRecords"),
```

Comment:
I know the phrasing of this message isn't part of this PR but I don't know if it might be a good idea for us to flag it. The phrasing doesn't quite make sense to me, don't know if you have any feelings about it too?

Reply:
Yeah, maybe "I confirm that in this version of TDR, all records that I **am transferring** are open" and/or maybe change "in this version of TDR" because it sounds ambiguous. We should probably let Kirsten know.

Reply:
Yeah the 'in this version of TDR' is the biggest thing throwing me off, but 'that I transfer' also doesn't sound completely smooth. I agree we should mention it to Kirsten :+1: